README.md

```diff
@@ -182,5 +202,18 @@ Currently, we are avoiding adding personal websites/blogs or services that do no
 
 ## Acknowledgements
 
-We would like to thank the team at **whotrack.me** for their work.
-Initially, our database was built on top of the **whotrack.me** database, using their extensive data collection. However, we would like to emphasise that our current database is now independent and updated separately from **whotrack.me**.
+We would like to thank the team at **whotracks.me** for their work.
+Initially, our database was built on top of the **whotracks.me** database, using their extensive data collection.
+However, we would like to emphasize that our current database is now independent
+and updated separately from **whotracks.me**.
+
+[dist/companies.json]: https://raw.githubusercontent.com/AdguardTeam/companiesdb/main/dist/companies.json
+[dist/trackers.json]: https://raw.githubusercontent.com/AdguardTeam/companiesdb/main/dist/trackers.json
+[dist/vpn_services.json]: https://raw.githubusercontent.com/AdguardTeam/companiesdb/main/dist/vpn_services.json
+[dist/whotracksme.json]: https://raw.githubusercontent.com/AdguardTeam/companiesdb/main/dist/whotracksme.json
+[source/companies.json]: https://raw.githubusercontent.com/AdguardTeam/companiesdb/main/source/companies.json
+[source/trackers.json]: https://raw.githubusercontent.com/AdguardTeam/companiesdb/main/source/trackers.json
+[source/vpn_services.json]: https://raw.githubusercontent.com/AdguardTeam/companiesdb/main/dist/vpn_services.json
+[source/whotracksme.json]: https://raw.githubusercontent.com/AdguardTeam/companiesdb/main/source/whotracksme.json
+[source/whotracksme_companies.json]: https://github.com/AdguardTeam/companiesdb/blob/main/source/whotracksme_companies.json
```

Comment:
should be in raw format too?
https://raw.githubusercontent.com/AdguardTeam/companiesdb/main/source/whotracksme_companies.json